Week2/js-exercises/ex1-oddOnesOut.js

```diff
@@ -7,15 +7,23 @@
  The function should still behave the same.
 
  */
+// function doubleEvenNumbers(numbers) {
+//   const newNumbers = [];
+//   for (let i = 0; i < numbers.length; i++) {
+//     if (numbers[i] % 2 === 0) {
+//       newNumbers.push(numbers[i] * 2);
+//     }
+//   }
+//   return newNumbers;
+// }
+
+// const myNumbers = [1, 2, 3, 4];
+// console.log(doubleEvenNumbers(myNumbers)); // Logs "[4, 8]" to the console
+
 function doubleEvenNumbers(numbers) {
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
-  return newNumbers;
+  let newNumber = numbers.filter((number) => number % 2 === 0).map((number) => number * 2);
```

Comment:
Great way applying the array functions! You could consider some indentation for readability of your code:
```suggestion
  let newNumber = numbers
      .filter(number => number % 2 === 0)
      .map(number => number * 2);
```
Or even directly return the complete expression without storing it in the `newNumbers` variable directly. But that are just optimizations :-)

Reply:
Thanks for your feedback and suggestions .. when i save after the indentation for readability the VSCode reverse the change how i can keep the line separated?

Reply:
Oh that might be a linter or a setting in VSCode. Don't worry about it for now. It takes time to tell what causes it and it it not that important.